spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -68,15 +68,18 @@ class SpotSkillManager:
         spotskillmanager.place("test_place_front")
     """
 
-    def __init__(self):
+    def __init__(self, should_stand_on_start: bool = True):
         # Create the spot object, init lease, and construct configs
         self.__init_spot()
 
         # Initiate the controllers for nav, gaze, and place
         self.__initiate_controllers()
 
         # Power on the robot
-        self.spot.power_robot()
+        if should_stand_on_start:
+            self.spot.power_robot()
+        else:
+            self.spot.power_on()
```

Comment:
@KavitShah1998 add comment here explaining what is this and why

Reply:
Discussed offline, we remove it

Reply:
removed